README.md

```diff
@@ -69,6 +69,7 @@ Current list of maintainers:
 
 * [Anthony Smith](https://github.com/anthonator)
 * [Igor Kapkov](https://igas.me)
+* [Gian D](https://github.com/fusillicode)
```

Comment:
this is the list of maintainers, not the contributors.

Reply:
Oh gosh thanks for pointing out! 🙇 

It should be fixed now! 😄